### DISCUSSION TITLE
Trivy Tutorial for CI/CD Semaphore

### DISCUSSION THREAD

**Discussion by TomFern** [aquasecurity/trivy#1352] [Adopters]
Hi! I'm finishing up a tutorial on using Trivy on a CI/CD for the [Semaphore Blog](https://semaphoreci.com/blog). Is there anyone here that would like to review it or offer advise. I can link the draft. Thank you 🙏

**Comment by knqyf263** [comment]
Thanks! I'll take a look. Could you share the draft with me?

**Comment by TomFern** [comment]
Hi @knqyf263. Thank you for the offer to review. I'm sending the link to the gmail address you have on your GitHub profile. Let me know if that's OK. 

Have a great day
Tomas

**Reply by TomFern** [reply]
Hey @knqyf263 did you get the link?

**Reply by knqyf263** [reply]
Yes, I did. I'll take a look this week.

**Comment by TomFern** [comment]
Tutorial published here: https://semaphoreci.com/blog/continuous-container-vulnerability-testing-with-trivy

Thank you @knqyf263 for helping me with the review 🙇

**Comment by JudeStrife** [comment]
Hello, I can't find the .trivy file ignore
Otherwise when I create it it is not taken into account 

Thanks for your help
